fortinet_fortigate/plugin.spec.yaml

```diff
@@ -627,26 +627,26 @@ connection:
     example: false
 
 actions:
-  check_if_ip_is_in_address_groups:
-    title: Check Address Group for IP Address
+  check_if_address_in_group:
+    title: Check if Address in Group
     description: Check if an IP address is in an address group
     input:
-      address_group_name:
-        title: Address Group Name
-        description: The name of the address group to check
+      group:
+        title: Group
+        description: Name of Address Group to check for address
         type: string
         required: true
         example: blocked IP's
-      ip_address:
-        title: IP Address
-        description: The IP address to check for
+      address:
+        title: Address
+        description: The IP, CIDR, or domain to check for
```

Comment:
Does this also accept address object by its name "MaliciousHost" ? If not, then the `example` input is wrong - that might be my fault

Reply:
Yes it accepts the name as well.